scaffold/gitignore/common.gitignore

```diff
@@ -28,7 +28,7 @@ web/sites/*
 web/sites/default/*
 !web/sites/default/settings.php
 !web/sites/default/services.yml
-!web/sites/default/services.*.yml
+!web/sites/default/*.services.yml
```

Comment:
The PR title says "ignore"... but this has a negation of `!`?

Reply:
Derp 😆 yes, thank you! Updated PR title